classes/user/maps/Schema.inc.php

```diff
@@ -0,0 +1,262 @@
+    public function summarizeReviewer(User $item, ?SlimRequest $slimRequest = null): array
+    {
+        return $this->mapByProperties(array_merge($this->getSummaryProps(), ['reviewsActive', 'reviewsCompleted', 'reviewsDeclined', 'reviewsCancelled', 'averageReviewCompletionDays', 'dateLastReviewAssignment', 'reviewerRating']), $item, $slimRequest);
+    }
+
+    /**
+     * Map a collection of Users
+     *
+     * @see self::map
+     */
+    public function mapMany(Enumerable $collection): Enumerable
+    {
+        $this->collection = $collection;
+        return $collection->map(function ($item) {
+            return $this->map($item);
+        });
+    }
+
+    /**
+     * Summarize a collection of users
+     *
+     * @see self::summarize
+     */
+    public function summarizeMany(Enumerable $collection): Enumerable
+    {
+        $this->collection = $collection;
+        return $collection->map(function ($item) {
+            return $this->summarize($item);
+        });
+    }
+
+    /**
+     * Map schema properties of a user to an assoc array
+     */
+    protected function mapByProperties(array $props, User $user, ?SlimRequest $slimRequest = null): array
+    {
+        $output = [];
+
+        foreach ($props as $prop) {
+            switch ($prop) {
+                case 'id':
+                    $output[$prop] = (int) $user->getId();
+                    break;
+                case 'userName':
+                    $output[$prop] = $user->getUserName();
+                    break;
+                case 'fullName':
+                    $output[$prop] = $user->getFullName();
+                    break;
+                case 'givenName':
+                    $output[$prop] = $user->getGivenName(null);
+                    break;
+                case 'familyName':
+                    $output[$prop] = $user->getFamilyName(null);
+                    break;
+                case 'affiliation':
+                    $output[$prop] = $user->getAffiliation(null);
+                    break;
+                case 'country':
+                    $output[$prop] = $user->getCountry();
+                    break;
+                case 'url':
+                    $output[$prop] = $user->getUrl();
+                    break;
+                case 'email':
+                    $output[$prop] = $user->getEmail();
+                    break;
+                case 'orcid':
+                    $output[$prop] = $user->getOrcid();
+                    break;
+                case 'biography':
+                    $output[$prop] = $user->getBiography(null);
+                    break;
+                case 'signature':
+                    $output[$prop] = $user->getSignature(null);
+                    break;
+                case 'authId':
+                    $output[$prop] = $user->getAuthId();
+                    break;
+                case 'authString':
+                    $output[$prop] = $user->getAuthStr();
+                    break;
```

Comment:
I know you didn't change anything here, but is it safe to have the `authId` and `authString` exposed in the API? Eventually non-managers will be able to access it. We might need some protections in the map (you can see how this is handled in the submission/publication map when data is passed to a reviewer).

Reply:
I think this old auth stuff should be removed in favour of a standard OpenID/OAuth implementation with whatever that entails. I recommend leaving this for now and cleaning it up either when we expose the user API more broadly, or when we integrate OpenID/OAuth support, whichever comes first.